app/core/src/main/java/com/fsck/k9/helper/MessageHelper.java

```diff
@@ -55,7 +55,7 @@ public CharSequence getSenderDisplayName(Address address) {
             return resourceProvider.contactUnknownSender();
         }
 
-        Contacts contactHelper = K9.isShowContactName() ? Contacts.getInstance(mContext) : null;
+        Contacts contactHelper = K9.isShowContactName() ? DI.get(Contacts.class) : null;
```

Comment:
Instead of doing the Koin lookup here, do it in `getInstance()` and inject via the constructor (like we already do with `CoreResourceProvider`).

Reply:
This class is going to be refactored in a follow up pull-request.